app/filters/currency-filter.js

```diff
@@ -16,7 +16,7 @@
       formats.DEFAULT_PRECISION = angular.isUndefined(formats.DEFAULT_PRECISION) ? 2 : formats.DEFAULT_PRECISION;
       return function ( amount, currencySymbol, fractionSize, suffixSymbol, customFormat ) {
         if ( currencySymbol != '$' ) { suffixSymbol = true }
-        if ( !angular.isNumber(amount) ) { amount = 0; }
+        if ( !angular.isNumber(amount) ) { amount = parseInt(amount, 10) }
```

Comment:
Using parseInt removes any decimals (since it converts to an integer). For example the first transaction 13-Nov-15 is $16,089.19 but is shown as $16,089.00. 

Even if we decide to go without cents going forward, we still need to be able to show historical tranactions correctly @jlopker

Reply:
ooo good catch! need to think of another solution..